ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml

```diff
@@ -122,7 +122,7 @@ tests:
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
       CONFIG_TYPE: minimal
       TEST_SUITE: experimental/reliability/minimal
-    workflow: openshift-e2e-openstack-ccm
+    workflow: openshift-e2e-openstack-ipi
```

Comment:
Should we rename the test?

Reply:
right, I will leave the default of that workflow to be run, which is conformance/parallel plus openstack specific tests.